userspace/libscap/scap.c

```diff
@@ -1246,6 +1246,18 @@ int scap_get_events_from_ppm_sc(IN uint32_t ppm_sc_array[PPM_SC_MAX], OUT uint32
 	return SCAP_SUCCESS;
 }
 
+int scap_native_id_to_ppm_sc(int native_id)
+{
+#ifdef __linux__
+	if (native_id < 0 || native_id >= SYSCALL_TABLE_SIZE)
+	{
+		return -1;
+	}
+	return g_syscall_table[native_id].ppm_sc;
+#endif
+	return -1;
```

Comment:
Put this return in an `#else` block so we don't get dead code floating around.
```suggestion
#else
	return -1;
#endif
```